src/App.js

```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+      <body className="App-header">
         <p>
```

Comment:
- [ ] hi, why are you using a `body` and a `p` tag here, if there is no specific reason to do this I suggest you remove these tags, as it is also giving error in the console please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/228336952-d429c8b2-63b0-4715-95cb-ee6f46277dbe.png)